helmfile.d/0420.grafana.yaml

```diff
@@ -0,0 +1,66 @@
+helmDefaults:
+  args:
+    - "--wait"
+    - "--recreate-pods"
+    - "--timeout=600" 
+    - "--force"
+    - "--reset-values"
+
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+releases:
+
+#######################################################################################
+## Grafana                                                                           ##
+#######################################################################################
+
+- name: "grafana"
+  namespace: "monitoring"
+  labels:
+    chart: "grafana"
+    component: "monitoring"
+    namespace: "monitoring"
+    vendor: "kubernetes"
+    default: "true"
+  chart: "stable/grafana"
+  version: "1.12.0"
+  values:
+    - "values/grafana.dashboards.yaml"
+    - persistence:
+        enabled: true
+        size: 10Gi
+        accessModes:
+          - ReadWriteOnce
+      datasources:
+        datasources.yaml:
+          apiVersion: 1
+          datasources:
+            - name: Prometheus
+              type: prometheus
+              url: 'http://kube-prometheus-prometheus:9090'
```

Comment:
make it a variable:
url:  {{ env "GRAFANA_PROMETHEUS_ENDPOINT" | default "http://kube-prometheus-prometheus:9090" }}'